src/group/libp2p_group_relcast_server.erl

```diff
@@ -408,13 +368,15 @@ lookup_worker(Key, KeyIndex, #state{workers=Workers}) ->
     lists:keyfind(Key, KeyIndex, Workers).
 
 -spec dispatch_ack(pos_integer(), pos_integer(), boolean(), boolean(), #state{}) -> #state{}.
-dispatch_ack(Index, Seq, Reset, Range, State=#state{}) ->
+dispatch_ack(_Index, [], _Reset, _Range, State=#state{}) ->
```

Comment:
Typespec is now wrong for dispatch_ack. Sould be a list of something

Reply:
fixed